docs/INSTALL.md

```diff
@@ -88,61 +269,7 @@ without the need for external interfaces.
 
 ![ubench-sim](images/ubench-sim.svg)
 
-To start UPF in simulation mode:
-
-1. Enable sim mode in configuration files
-
-    ```patch
-    diff --git a/conf/upf.json b/conf/upf.json
-    index 15042f9..e5a4588 100644
-    --- a/conf/upf.json
-    +++ b/conf/upf.json
-    @@ -2,7 +2,7 @@
-        "": "Vdev or sim support. Enable `\"mode\": \"af_xdp\"` to enable AF_XDP mode, or `\"mode\": \"af_packet\"` to enable AF_PACKET mode, or `\"mode\": \"sim\"` to generate synthetic traffic from BESS's Source module",
-        "": "mode: af_xdp",
-        "": "mode: af_packet",
-    -    "": "mode: sim",
-    +    "mode": "sim",
-
-        "": "max UE sessions",
-        "max_sessions": 50000,
```

Comment:
sim mode is good for exploration, one can play with parameters like max_session, tables_sizes and immediately verify upf functioning in standalone mode without requiring hardware/aiab etc. we shall mention these capabilities to users who want explore upf from their point of view

Reply:
Well, this document is supposed to be about `installation`. I am going to try to add some lines about your comment in the installation documentation